src/routes/gallery/lib/gallery.ts

```diff
@@ -58,18 +59,23 @@ export const getImagesFromWNFS: () => Promise<void> = async () => {
 
         // The CID for private files is currently located in `file.header.content`,
         // whereas the CID for public files is located in `file.cid`
-        const cid = isPrivate ? file.header.content.toString() : file.cid.toString()
+        const cid = isPrivate
+          ? (file as any).header.content.toString()
+          : (file as any).cid.toString()
 
         // Create a base64 string to use as the image `src`
-        const src = `data:image/jpeg;base64, ${uint8arrays.toString(file.content, 'base64')}`
+        const src = `data:image/jpeg;base64, ${uint8arrays.toString(
+          (file as any).content,
+          'base64'
+        )}`
 
         return {
           cid,
-          ctime: file.header.metadata.unixMeta.ctime,
+          ctime: (file as any).header.metadata.unixMeta.ctime,
```

Comment:
A bit better to have `any` here instead of ignoring the errors. 👌 

Not a request for this PR, but we may want to define some custom interfaces for the types we would like to see and `as` to them . That would provide a richer interface for folks reading the code, and it would inform the interface design when we rework these type in Webnative.

Reply:
I ended up making a custom `GalleryFile` type 👌🏼